### DISCUSSION TITLE
GTD capture without `:NeorgStart`

### DISCUSSION THREAD

**Discussion by pca006132** [nvim-neorg/neorg#486] [Q&A] [answered]
Hi! I'm wondering if it is possible to use GTD capture without calling `:NeorgStart` first, as I often have some ideas that I want to note down while writing some other stuff, calling `:NeorgStart` requires another command which lengthens the critical path, and changes the current directory which is quite annoying when editing other files.

**Comment by max397574** [comment, marked as answer]
it isn't possible
you can do `NeorgStart silent=true` to not change your directory

**Reply by molleweide** [reply]
is it possible to set silent as default in config?

**Reply by max397574** [reply]
I don't think so
but it should be possible
@vhyrro

**Reply by arsham** [reply]
You can start by creating a new window (`:vert new` for instance), and on the new window do `:NeorgStart`. Because this plugin uses `lcd` for changing the working directory, your other windows should retain their working directory.
